Dockerfile

```diff
@@ -63,26 +63,26 @@ COPY package.json /temp/prod/package.json
 # Install every prod dependencies
 RUN --mount=type=cache,id=pnpm,target=/pnpm/store cd /temp/prod && pnpm install --prod --frozen-lockfile
 
-# Final stage
-FROM debian:stable AS final
-
-WORKDIR /root
+# Create symlink stage
+FROM alpine:latest AS symlink
+RUN mkdir -p /root && ln -s /erpc-server /root/erpc-server
 
-# Install CA certificates
-RUN apt-get update --allow-insecure-repositories \
-    && apt-get install -y debian-archive-keyring ca-certificates \
-    && rm -rf /var/lib/apt/lists/*
+# Final stage
+FROM gcr.io/distroless/static-debian12:nonroot AS final
 
 # Copy Go binary from go-builder
-COPY --from=go-builder /root/erpc-server .
-COPY --from=go-builder /root/erpc-server-pprof .
+COPY --from=go-builder /build/erpc-server /
+COPY --from=go-builder /build/erpc-server-pprof /
+
+# Copy symlinked directory
+COPY --from=symlink /root /root
```

Comment:
This doesn't seem correct, we need a symlink on /root/erpc-server -> /erpc-server in the final image